### PR TITLE
Configure TickSpec build for AppVeyor CI

### PR DESCRIPTION
**Pull request by mchaloupka**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,176 @@
+                                 Apache License
```
Comment (bartelink) [inline LICENSE]: @ptrelford what should the license be ?

File: build.bat [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
-cd ..\..
\ No newline at end of file
+msbuild TickSpec.sln /p:Configuration=Release /m
+
+if errorlevel 1 (
+  echo "Build failed"
+  exit /b %errorlevel%
+)
+
+if %1.==. (
```
Comment (bartelink) [inline build.bat]: can you quote the comparands please ?

File: build.bat [hunk 3]
```diff
@@ -1,20 +1,35 @@
...
+    set version="1.0.1.1"
+) else (
+    set version="%1"
+)
+
+.paket\paket.exe pack packed_nugets --version %version% --symbols
+
+pushd %~dp0packed_nugets
+%NuGetTool% pack ..\Nuget\NUnit\TickSpec.NUnit.nuspec -Version %version%
+%NuGetTool% pack ..\Nuget\xUnit\TickSpec.xUnit.nuspec -Version %version%
```
Comment (bartelink) [inline build.bat]: Maybe we should drop this - few people use xUnit v1 and it's not helpful to send people the wrong way (we might need to unlist the package ... or maybe we should log an issue to do a package that has all the helpers ready to go for xUnit v2 without the sample - obviously this was all done before Paket so for example there are source files in the package which are intended to be edited, which is not a good pattern)

File: LICENSE (hunk 1 above)
Comment (mchaloupka) [reply LICENSE]: I took it from here: http://tickspec.codeplex.com/license

Comment (bartelink) [reply LICENSE]: Hm, how did I drop it, apologies!

**Comment by bartelink** [comment]
Feel free to merge when you're ready (I prefer not to squash  if the individual commits are clean)

File: build.bat (hunk 2 above)
Comment (mchaloupka) [reply build.bat]: Done.

File: build.bat (hunk 3 above)
Comment (mchaloupka) [reply build.bat]: I commented it out. We should probably consider how to provide such thing (does it really need a nuget)? For example, the NUnit variant will not work even for NUnit3. I agree that handling it as a separate issue is good way to go.

File: LICENSE (hunk 1 above)
Comment (mchaloupka) [reply LICENSE]: I have just noticed it when converting the nuspec to paket.template. I also fixed the link to the project there.

File: build.bat (hunk 3 above)
Comment (bartelink) [reply build.bat]: Yeah - if you look on the DI branch, I have a cleaner way to to the NUnit linking (based onthe xUnit style) which does not bind to the specific version of NUnit (i.e. no base type).
Will focus on getting DI merged first but happy to discuss the approach on an Issue if you want

**Pull request merged**